src/cljs/chronograph_web/events/routing.cljs

```diff
@@ -1,11 +1,34 @@
 (ns chronograph-web.events.routing
   (:require [re-frame.core :as rf]
-            [chronograph-web.db :as db]))
+            [chronograph-web.http :as http]
+            [chronograph-web.pages.organization.events :as org-events]))
+
+
+(defn- make-get-organization-uri
```

Comment:
```suggestion
(defn- get-organization-uri
```

Reply:
Also this should move out of this namespace along with the GET request.

Reply:
Done, moved to the org-events ns.